src/components/menu/js/menuDirective.js

```diff
@@ -258,6 +258,7 @@ function MenuDirective($mdUtil) {
   function link(scope, element, attr, ctrls) {
     var mdMenuCtrl = ctrls[0];
     var isInMenuBar = !!ctrls[1];
+    var mdMenuBarCtrl = ctrls[1];
```

Comment:
What's the value when not in a menu bar? `null`? `undefined`? If that's the case, then I suppose this is fine in that case, but I want to be sure that it's not something else.

Reply:
It appears to be `null` in that case.